public/style/components/_data.scss

```diff
@@ -0,0 +1,10 @@
+.data-primary {
+  @extend .large;
+  margin-bottom: 0;
+  margin-top: .3em;
+}
+
+.data-secondary {
```

Comment:
I'm wondering if this should even be a class anymore. Places where `.data-secondary` is used can just have both the class `.small` and `.muted`